afpacket/src/sockets.rs

```diff
@@ -139,6 +142,24 @@ impl Socket {
 }
 
 impl BoundSocket {
+    /// Puts this socket's NIC into promiscusous mode. Useful for recieving all packets on an
+    /// interface, including those not addressed to the device.
+    pub fn set_promisc(&mut self) -> io::Result<()> {
```

Comment:
`promisc` almost looked like `promise` to me. Maybe we can expand the function name to `set_promiscuous` for clarity?

Reply:
Should we have an option to toggle promiscuous mode on and off?

Reply:
Resolved the name.